src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/tests/recordings/batch_help_full.txt

```diff
@@ -116,11 +115,10 @@ Global Arguments
                                     information and examples.
     --verbose                     : Increase logging verbosity. Use --debug for full debug logs.
 
-(env) C:\Code\Forks\azure-cli>cmd /c az batch account set --help 
+(env) D:\GitHub\azure-cli\src\command_modules\azure-cli-batch\azure\cli\command_modules\batch\tests>cmd /c az batch account set --help 
 
 Command
-    az batch account set: Update the properties of the specified Batch account. Properties that are
-    not specified remain unchanged.
```

Comment:
I'm a little unsure as to why the suffix ('Properties that are not specified remain unchanged.') has been lost on the 'set' commands - I think this might be another parsing issue like the missing 'destination'. I'll take a look to remind myself under what circumstances this was appended.

Reply:
Ignore this - the help file was updated out of band for consistency across command modules.